### DISCUSSION TITLE
bolts question

### DISCUSSION THREAD

**Discussion by TheHammerMan00** [makerforgetech/modular-biped#63] [Q&A] [answered]
I have the legs printed and the servos/ bearings are on order.  I haven't been to the local hardware store yet but what bolts are you guys using?  7.5mm seems to be an oddball size.  I'm not above taking my calipers to the store but thought I would just ask first.

**Comment by WhoIsMrSentry** [comment]
Hello, first of all, welcome to the discussion. In my own version, I bought parts such as bolts, screws  salmon and bearings with a 3D printer. If you cannot find them, I can recommend them to you.

**Comment by danic85** [comment, marked as answer]
I'm using bolts that I had already, but really they just need to have a diameter that is close to the internal diameter of your bearings. If they're too loose you'll have a bit of play in them (or the bearings won't be as effective), and obviously too tight won't fit.

**Comment by Chompalot** [comment]
Those will be M8, if it's for the skate board bearings
